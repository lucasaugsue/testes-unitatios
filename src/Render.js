import React from 'react';
// import Header from '../src/screens/Header';
// import TabelaFinancias from './screens/TabelaFinancias';

function Render() {

  return (
    <div>
        os componentes aqui
        {/* <Header/> */}
        {/* <TabelaFinancias/> */}
    </div>
  );
}

export default Render;
