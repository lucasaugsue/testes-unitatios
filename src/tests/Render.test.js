import renderer from 'react-test-renderer';
import Render from '../Render';

// Caso de teste atualizado com um Link para um endereÃ§o diferente
it('renderiza corretamente', () => {
    const tree = renderer
      .create(<Render/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });