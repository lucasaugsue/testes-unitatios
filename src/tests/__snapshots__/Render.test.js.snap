// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderiza corretamente 1`] = `
<div>
  <div>
    <div
      className="basicHeader"
    >
      <div
        className="gridContainerLogo"
      >
        <div
          className="itemLogo"
        >
          <div
            className="logo"
          >
            S
          </div>
          <div
            className="logo"
          >
            e
          </div>
          <div
            className="logo"
          >
            n
          </div>
          <div
            className="logo"
          >
             
          </div>
          <div
            className="logo"
          >
            F
          </div>
          <div
            className="logo"
          >
            i
          </div>
          <div
            className="logo"
          >
            n
          </div>
          <div
            className="logo"
          >
            a
          </div>
          <div
            className="logo"
          >
            n
          </div>
          <div
            className="logo"
          >
            รง
          </div>
          <div
            className="logo"
          >
            a
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
